core/templates/dev/head/components/CircularImageDirective.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Displays circled images with linking (when available).
+ *
+ * @author 526avijit@gmail.com (Avijit Gupta)
+ */
+
+oppia.directive('circularImage', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      picture: '&',
```

Comment:
@seanlip  I had thought of using `src`, but then pivoted away from it since even the default `<img>` tag uses it. (`<img src="" alt=""/>)` . So just to avoid confusion.
 Let's use `imageSrc`?